libs/ui-watt/src/lib/components/input/_variables.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+$placeholder-color: var(--watt-color-neutral-grey-500);
+$input-height-normal: 42px;
```

Comment:
question: the value here is 2px less than what it was before. Is this intentional? If yes, this will cause an issue in places where there's a button next to an input, because the button has `height` set to 44px.

Reply:
I think the 1px border on each side should align the input with the buttons

Reply:
Oh, sorry. The changes are related to date- and time- pickers. I confused it with input in`watt-form-field`. It seems like 42px doesn't make a difference.

![image](https://user-images.githubusercontent.com/1096332/214307248-2fd699ce-ad5e-4e60-9eaa-312ad61aef7d.png)

Reply:
yes actually, it seems it's been set [here ](https://github.com/Energinet-DataHub/greenforce-frontend/blob/main/libs/ui-watt/src/lib/components/form-field/form-field.component.scss#L78), which confuses me a little as to why we're experiencing #1034, 

I'll convert it to draft and investigate a little further